content/posts/hello-world-dbanalytics.md

```diff
@@ -1,13 +1,45 @@
+title: "Hello, World: Becoming a Freelance Consultant"
 description: "Why I'm moving into freelance consulting."
-draft: true
+draft: false
 ---
 
-At the end of next week, I will leave my full-time job as BI Lead at Growth Street. 
+At the end of this week, I'll be leaving my full-time job as BI Lead at Growth Street. I'm incredibly excited to be making the move into freelance consulting.
 
+I'm writing this quick post to talk a bit about why I'm making the move, as much for my own benefit as that of the three other people who will read this blog (hi mum 👋).
 
-![Hope this works!](/test-image.jpg)
+## Why Now
 
+The analytics toolkit available to early-stage businesses has completely changed over the past five years. Five years ago, building out an analytics function was an expensive and difficult undertaking. [It required a fair bit of data engineering](https://blog.fishtownanalytics.com/does-my-startup-data-team-need-a-data-engineer-b6f4d68d7da9). It required building a lot of things from the ground up, well before you saw any value from it.
+
+That has completely changed. The advent of tools like Stitch, Fivetran, dbt, and Snowflake means you can now get up and running very quickly and incredibly cost-effectively. Moreover, you'll have built something that is equipped to scale as your business does. I've spent the last few years [setting up these data stacks in-house for a handful of VC-backed businesses](https://www.growthstreet.co.uk/blog/product/growth-street-bi-stack).
+
+My plan is to work with high-growth businesses, helping them start or scale their use of analytics, putting the right building blocks in place and helping them get value from all the data that is at their disposal. 
+
+I'm really excited about the first few businesses that I'll be working with. If you're interested in having a chat about how your business might be able to make better data-driven decisions, [get in touch!](/contact)
+
+## Moving to Canada 🇨🇦 
+
+A huge factor in my decision is also the fact that I'll be moving to Canada this summer. I moved to the UK from Montreal when I was 13 and am excited to be moving back for a few years. I'll be based in Toronto from later this year.
+
+With the move came a desire for increased flexibility. Freelancing ticked a lot of those boxes. It will hopefully allow me to take a lot of my remote work with me and giving me the time to do a bit of travelling around North and South America from our base in Toronto.
+
+## Writing more
+
+I hope that this post will be the first of many this year. I haven't written much during my time at Growth Street and I've really missed it.
+
+The plan is to write about both the technical and strategic aspects of analytics. My first post will cover the trade-offs between Looker's PDTs and the open-source tool dbt; discussing the reasons why you may want to consider the latter over the former. If that's of interest to you, keep an eye out for it. You can sign up to receive posts via email below.
```

Comment:
The primary CTA, "keep an eye out for it", is a bit meh here – I'd make "sign up" the primary CTA. Something like:
>>>  If that's of interest to you, sign up below to have it delivered straight to your inbox, or keep an eye on this page for it.